.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- [ ] Please branch name should describe a feature like feature/(actionName)
- [ ] Please provide a brief good description of the pull request.
![image](https://user-images.githubusercontent.com/74744787/185756696-f204dade-6b67-4715-b8bf-4f2de8aa4955.png)